apiserver/cert_test.go

```diff
@@ -5,6 +5,7 @@ package apiserver_test
 
```

Comment:
Please fix the PR title to indicate the area of the code - it's a bit generic.

Something like: `apiserver: Loosen up error checks`